### PR TITLE
update: link-only mode for dynamic frameworks

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

File: docs/topics/native/native-cocoapods-dsl-reference.md [hunk 1]
```diff
@@ -119,15 +119,17 @@ kotlin {
 The `pod()` function call adds a CocoaPods dependency to the Pod built from this project. Each dependency requires
 a separate function call.
 
-You can specify the name of a Pod library in the function parameters and additional parameter values, like the `version` and
-`source` of the library, in its configuration block.
+In the function parameters, you can specify the name of a Pod library and the `linkOnly` option to use pod dependencies
```
Comment (daugeldauge) [inline docs/topics/native/native-cocoapods-dsl-reference.md:122]: It's also available in the configuration block. So I would add it to the table bellow. Actually all parameters apart from name of the pod are configurable in it.

Comment (danil-pavlov) [reply docs/topics/native/native-cocoapods-dsl-reference.md:125]: Thank you! moved to the table

File: docs/topics/native/native-cocoapods-dsl-reference.md [hunk 2]
```diff
@@ -141,6 +147,7 @@ kotlin {
       
         pod("pod_dependency") {
             version = "1.0"
+            linkOnly = "true"
```
Comment (daugeldauge) [inline docs/topics/native/native-cocoapods-dsl-reference.md:150]: Without quotes :)

**Review by daugeldauge — APPROVED** [review]

File: docs/topics/native/native-cocoapods-dsl-reference.md [hunk 3]
```diff
@@ -118,15 +118,21 @@ kotlin {
...
-|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `version`     | The library version. To use the latest version of the library, omit the parameter.                                                                                                                                                                                                                                                                          |
+In the function parameters, you can specify the name of a Pod library and the `linkOnly` option to use pod dependencies
+with dynamic frameworks only for linking without generating C-interop bindings.
+
+In the configuration block, you can add parameter values, like the `version` and `source` of the library:
```
Comment (sarahhaggarty) [inline docs/topics/native/native-cocoapods-dsl-reference.md:127]: Is this a duplicate of line 122?

File: docs/topics/native/native-cocoapods-dsl-reference.md [hunk 4]
```diff
@@ -118,15 +118,21 @@ kotlin {
...
-|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `version`     | The library version. To use the latest version of the library, omit the parameter.                                                                                                                                                                                                                                                                          |
+In the function parameters, you can specify the name of a Pod library and the `linkOnly` option to use pod dependencies
+with dynamic frameworks only for linking without generating C-interop bindings.
```
Comment (sarahhaggarty) [inline docs/topics/native/native-cocoapods-dsl-reference.md:125]: I think it's a bit easier to understand this sentence if we rearrange it to explain the WHAT before the HOW.
```suggestion
To use pod dependencies with dynamic frameworks for linking only, without generating C-interop bindings, specify the name of a Pod library and the `linkOnly` option in the function parameters.
```

**Review by sarahhaggarty — CHANGES_REQUESTED** [review]
Just one thing to check and a suggestion 😃

File: docs/topics/native/native-cocoapods-dsl-reference.md (hunk 3 above)
Comment (danil-pavlov) [reply docs/topics/native/native-cocoapods-dsl-reference.md]: True, I removed the duplicated text and simplified the whole desciption

**Review by sarahhaggarty — APPROVED** [review]

**Pull request merged**